remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/notifications/buildTask/BuildStatusMQNotifications.java

```diff
@@ -49,7 +50,7 @@ public BuildStatusMQNotifications(MessageSenderProvider messageSenderProvider) {
         this.messageSender = messageSenderProvider.getMessageSender();
     }
 
-    public void observeEvent(@Observes BuildStatusChangedEvent event) {
+    public void observeEvent(@Observes(during = TransactionPhase.AFTER_COMPLETION) BuildStatusChangedEvent event) {
```

Comment:
Why AFTER_COMPLETION and not AFTER_SUCCESS? If the transaction rolls back, why do we want to sent the UMB message? This part of the code could be (very likely) retried and _may_ succeed which could send a second UMB message.

Reply:
That would mean:
- [this](https://github.com/project-ncl/pnc/blob/529413ceea6fc741647428c2c26e965153bfbe18/remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/RemoteBuildCoordinator.java#L629) does not neet to be in finally (since on error it will not be observed)
- [this](https://github.com/project-ncl/pnc/blob/529413ceea6fc741647428c2c26e965153bfbe18/remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/RemoteBuildCoordinator.java#L742) could be removed (since on error it will not be observed)